docs/src/pages/landing-page/Index.vue

```diff
@@ -226,13 +213,45 @@ export default defineComponent({
   border-bottom: 1px solid $lp-light;
 }
 
-.support {
-  height: 100vh;
-}
-
 .social-channels-call-to-action {
   // TODO: reduce image height (from top), and remove this negative margin
   margin-top: -30%;
 }
 
+.why-quasar-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 16px;
+  margin: 0 auto;
+  width: 80%;
+}
+
+.astronaut-hand {
+  z-index: 0; // ensure image is always displayed behind text
+  width: 53%;
+  position: absolute;
+
+  &--left {
+    left: 0;
+    bottom: 0;
+
+    @media screen and (max-width: $breakpoint-sm-max) {
```

Comment:
When working with responsiveness, most of the time it's easier (and produce a cleaner code) to start with the smaller viewport rules, then add the ones for the bigger one
This is because the structure and CSS on mobile are usually much simpler than desktop ones

As a rule of thumb, always start small and grow bigger (so, using `min-width` instead of `max-width`). Only do the opposite if you see the resulting code is ugly abd would result much cleaner and simpler doing the other way around

Reply:
Roger that, the issue here is that I started with desktop first. Hence, it became easier to use max-width instead of min-width later on. Using min-width will require me to change the styles to mobile first and then adapt to desktop. I have tried doing this for some but there was no much benefit, so I continued with max-width. Of course, I have kept your advice in mind, but if you think it has to be changed then let me know and I will update the styles first to mobile and then adapt to desktop.

Reply:
We often start from desktop, usually due to time constraint and it being the most used viewport
When we do, we usually get back and refactor the code later on to actually "start" from mobile
Of course the more skilled you are, the more you can do this from the beginning

Since we still have some time, go on and refactor styles to start from mobile and grow to desktop

Reply:
okay, sure.